tests/functional/test_to_idp_interstitial.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from flask import url_for
+import pytest
+
+
+@pytest.mark.usefixtures('live_server')
+class When_on_to_idp_interstitial(object):
+
+    @pytest.fixture(autouse=True)
+    def setup_page(
+            self,
+            browser,
+            email_address,
+            submit_known_email_address):
+        submit_known_email_address(email_address)
+        browser.find_by_css('form button').click()
+
+    def it_goes_to_the_idp_after_clicking_continue(
+            self, browser, config, responses):
+
+        browser.find_by_css(
+            '#content>div.pagination>ul>'
+            'li>a>span.pagination-text>span').click()
```

Comment:
This line is broken at an arbitrary point. Break lines either after an opening bracket/parenthesis or at 80 columns (or below if a word crosses the 80th column).

This CSS selector is probably unnecessarily long - do you need the `div` in `div.pagination`? CSS selectors should be as specific as necessary, but no more. What if the tag with the `pagination` class is changed to a `p`, for example.

This selector could probably be reduced `.pagination .next a` - the `a` is the element that handles the `click` event, anyway.